client/build/asset-manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "files": {
+    "main.css": "/static/css/main.964fdded.css",
+    "main.js": "/static/js/main.100d7049.js",
+    "index.html": "/index.html",
+    "main.964fdded.css.map": "/static/css/main.964fdded.css.map",
+    "main.100d7049.js.map": "/static/js/main.100d7049.js.map"
+  },
+  "entrypoints": [
+    "static/css/main.964fdded.css",
+    "static/js/main.100d7049.js"
+  ]
+}
```

Comment:
In general it's a good idea too add the build folder to the git ignore file, so that these bit's don't clatter your source code.

Reply:
Thank you!